src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard-v3/dashboard/dashboard-v3.component.html

```diff
@@ -31,12 +31,18 @@
           </dd>
           <ng-container>
             <dt>Telemetry Dashboard
-              <span
-                class="badge"
-                [ngClass]="telemetryEnabled ? 'badge-success' : 'badge-secondary'"
-                [ngbTooltip]="getTelemetryText()" >
-                {{ telemetryEnabled ? 'Active' : 'Inactive' }}
-              </span>
+              <cds-tag *ngIf="telemetryEnabled"
+                       class="tags-success"
+                       [size]="sm"
+                       [ngbTooltip]="getTelemetryText()">
+                Active
+              </cds-tag>
+              <cds-tag *ngIf="!telemetryEnabled"
+                       class="tags-secondary"
+                       [size]="sm"
+                       [ngbTooltip]="getTelemetryText()">
+                Inactive
+              </cds-tag>
```

Comment:
No need to add two different `<cds-tag>` components. 

Instead you can render the component conditionally as it was being done before.

Also, `[size]="sm"` is not being picked up as is trying to access an `sm` variable, which does not exist in the component. Instead, you would need to do `size="sm"` to apply the `sm` size.
However I would advice to use the default medium size for all the tags. So we can completly remove this `[size]="sm"`

Reply:
done